api/server/router/swarm/cluster_routes.go

```diff
@@ -167,7 +167,16 @@ func (sr *swarmRouter) getServices(ctx context.Context, w http.ResponseWriter, r
 		return errdefs.InvalidParameter(err)
 	}
 
-	services, err := sr.backend.GetServices(basictypes.ServiceListOptions{Filters: filter})
+	var status bool
+	if value := r.URL.Query().Get("status"); value != "" {
```

Comment:
Can you gate this by API version, so that setting this query-parameter does not have an effect on older API versions?